.github/workflows/make-release.yml

```diff
@@ -38,6 +41,41 @@ jobs:
         openssl sha1 -r "$GITHUB_REF_NAME.tar.gz" > "$GITHUB_REF_NAME.tar.gz.sha1"
         openssl sha256 -r "$GITHUB_REF_NAME.tar.gz" > "$GITHUB_REF_NAME.tar.gz.sha256"
         gpg -u "$SIGNING_KEY_UID" -o "$GITHUB_REF_NAME.tar.gz.asc" -sba "$GITHUB_REF_NAME.tar.gz"
+    - name: "Build and validate SBOM against git tree"
```

Comment:
I believe the validation must be done before we push the tag. Perhaps make it a part of [release-tools](https://github.com/openssl/tools) scripts. Besides, did you consider using https://github.com/anchore/sbom-action

Reply:
Why before?